db/migrations/20180207111044_create-meal-foods-table.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function(knex, Promise) {
+  let createQuery = `CREATE TABLE meal_foods(
+    id SERIAL PRIMARY KEY NOT NULL,
+    meal_id INTEGER,
```

Comment:
I know this works after working with you on generating the sequel query, but I'm wondering if this is the most effective way. Previously you mentioned you had REFERENCES instead of INTEGER and I'm curious to know if well be running into some problems/difficulties later in development due to choosing one over the other.

Reply:
Great question that I'm not sure about.

I had tried "meal_id INTEGER references meals, food_id INTEGER references foods" that ran into some foreign_key restraints when truncating the tables to reseed each time on the tests.